components/canvas/raqote_backend.rs

```diff
@@ -183,8 +184,28 @@ impl Path {
         unimplemented!()
     }
 
-    pub fn contains_point(&self, _x: f64, _y: f64, _path_transform: &Transform2D<f32>) -> bool {
-        unimplemented!()
+    pub fn contains_point(&self, x: f64, y: f64, _path_transform: &Transform2D<f32>) -> bool {
+        for op in self.as_raqote().ops.iter() {
```

Comment:
I'm just curious is it possible to run this with [`any`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.any) with iterator 👀?

```rust
self.as_raqote().ops.iter().any(|op| match op {
  ...
})
```

Reply:
I did not know about `any`. It's exactly what I intended to do, so thanks for pointing it out!